public/lib/inverted-index-spec.js

```diff
@@ -0,0 +1,193 @@
+// const values = require('object.values');
+
+//  book with valid contents
+const InvertedIndexClass = require('../../app/class/invertedIndexClass');
```

Comment:
Thanks.
In my test configuration (Jasmine SpecRunner and my-dexit.conf.js), I added a link to the class file before adding the link to the test spec, that way it two files can access each others' properties without having to export the class.